src/sql/workbench/contrib/notebook/browser/notebookEditor.component.html

```diff
@@ -4,6 +4,7 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 -->
+<loading-spinner class="editorLoader" *ngIf="isLoading" [loading]="isLoading"></loading-spinner>
```

Comment:
Please make sure you test this for screen reader support too - we should be making sure it announces when loading is started and done (with appropriate descriptions, such as "Loading Notebook content")

@alanrenmsft

Reply:
Would that just be the loadingMessage and loadingCompletedMessage properties on the loading spinner?

Reply:
adding loadingMessage and loadingCompletedMessage might be enough, because you have ngIf, so I am not sure whether it will announce the changes, because it is created and removed when isLoading changes. I think you should be able to remove the ngIf attribute. please give it a try.

Reply:
@alanrenmsft I added the loading messages, but I can't tab into the loading spinner to get them to be narrated. Are there any other examples I can look at where we've made accessibility changes to a loading spinner?

Reply:
The loading messages should be announced automatically. Are you not seeing that when the page loads?

Reply:
Never mind, I get the loading start message. I think I was opening the narrator too late.

Reply:
Ok nevermind, it turns out that the loading component isn't using the accessible loading spinner https://github.com/microsoft/azuredatastudio/blob/c89aa26c0a251a6b172e4542da268e513e9343c7/src/sql/base/browser/ui/loadingSpinner/loadingSpinner.ts

@alanrenmsft Do we not have an angular component using that?

Reply:
Oh ok, looked at it and while we're not using the loading spinner component we have the `status` call in setProperties. So little bit of duplicated effort here but it should work.

Reply:
Should I put the file name in the loading message, like "Loading notebook Notebook-0", or is "Loading notebook" by itself fine?

Reply:
Let's just do Loading notebook for now, it should hopefully already be announcing the editor title

Reply:
It only does for the first loading message. If you open a bunch of notebooks while they're still loading, then the completion message is ambiguous.

Reply:
That's weird - so if you switch editors it doesn't announce the new Notebook?

Anyways, I'm fine adding the name in there - doesn't really hurt to be a little extra verbose and may avoid confusion.

Reply:
It announces the editor, but you can get loading completed messages from editors that aren't in focus.